src/main/java/lotto/domain/LottoTicketBundle.java

```diff
@@ -0,0 +1,44 @@
+package lotto.domain;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+public class LottoTicketBundle {
+    private final List<Lotto> tickets;
+
+    private LottoTicketBundle(List<Lotto> tickets) {
+        this.tickets = List.copyOf(tickets);
+    }
+
+    public static LottoTicketBundle from(LottoGenerateStrategy strategy, int count) {
+        List<Lotto> tickets = IntStream.range(0, count)
+                .mapToObj(i -> new Lotto(strategy.pickNumbers()))
```

Comment:
i를 쓰지 않는데 ()으로 대체하는 거 어떠신가요

Reply:
IntStream의 mapToObj 메서드는 IntFunction<R>이라는 함수형 인터페이스를 요구하며, 이 인터페이스는 int 값을 인자로 받는 메서드를 구현해야 합니다. 이에 따라 위와 같이 구현했습니다 😄 

```java
/**
 * Represents a function that accepts an int-valued argument and produces a
 * result.  This is the {@code int}-consuming primitive specialization for
 * {@link Function}.
 *
 * <p>This is a <a href="package-summary.html">functional interface</a>
 * whose functional method is {@link #apply(int)}.
 *
 * @param <R> the type of the result of the function
 *
 * @see Function
 * @since 1.8
 */
@FunctionalInterface
public interface IntFunction<R> {

    /**
     * Applies this function to the given argument.
     *
     * @param value the function argument
     * @return the function result
     */
    R apply(int value);
}
```